Sources/Core/Lifetime.swift

```diff
@@ -0,0 +1,10 @@
+// Copyright © 2022 Max Desiatov <max@desiatov.com>.
+// All Rights Reserved.
+// SPDX-License-Identifier: BSD-3
+
+@inlinable
+public func withUnsafeMutablePointer<T, Result>(to value: inout T,
+                                                _ body: (UnsafeMutablePointer<T>) throws -> Result)
```

Comment:
Is this the formatting style you prefer for functions with long parameter lists? This exceeds 80 columns though.

Reply:
Yeah, when there is no way to break it up, then yes :-(  In such a case, prefer to break at the nearest point (which happens to be after the `)` in this case.